### DISCUSSION TITLE
Headings

### DISCUSSION THREAD

**Discussion by thewhaleiam** [awcodes/filament-tiptap-editor#413] [Q&A] [answered]
I can see that the editor supports level 1 through 6 for headings, can this be defined granularly in the config so we can define which heading are available to the editor, for e.g if you only want to support h1 and h2.

**Comment by awcodes** [comment, marked as answer]
Currently, this isn't possible.

**Comment by zakmcrbit** [comment]
A dirty workaround is to paste this into your Filament Backend theme.css:

.tiptap-tool[x-tooltip="'Heading 4'"],
.tiptap-tool[x-tooltip="'Heading 5'"],
.tiptap-tool[x-tooltip="'Heading 6'"]{
	display: none !important;
}
